src/components/AdminButton.jsx

```diff
@@ -0,0 +1,18 @@
+import { useContext } from "react";
+import { MdLibraryAdd } from "react-icons/md";
+import AuthContext from "../store/auth-context";
+
+export function AdminButton(props) {
+  const auth = useContext(AuthContext);
+  const { setAddModus, onClick: onClickHandler, label } = props;
```

Comment:
Is die onClickHandler een React ding? Want ik zie hem nergens beschreven?

Reply:
Dat antwoord moet ik je nog even verschuldigd blijven. Het idee was om het hele component een onClick mee te geven, in plaatst van alleen aan het icoontje. Dat is nog niet gelukt.